ansible/library/droplet_vlan_cfg.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+
+import re
+import yaml
+import os
+import traceback
+import subprocess
+import ipaddr as ipaddress
+from operator import itemgetter
+from itertools import groupby
+from collections import defaultdict
+
+DOCUMENTATION = '''
+module: droplet_vlan_cfg.py
+Ansible_version_added:  2.0.0.2
+short_description: Gather all vlan info related to droplet interfaces 
+Description:
+       When deploy testbed topology with droplet connected to TOR SONiC, gather interface aliases info for generating SONiC minigraph file
```

Comment:
This description doesn't seem to be correct, and it is also not worded clearly. Please fix it up.

Reply:
> I'd better to put vlan configuration into topologies.

I view the vlan to be orthogonal to the underlying topology. If someone would like to have 2, 3, or 4 vlans with T0, this is down by adding 4 configuration file. These configuration could be combined with different T0 topos.